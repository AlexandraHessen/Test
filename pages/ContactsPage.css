.ContactsPage{
    padding: 15px;
    background-color: #fff;
}

.ContactsPage h3{
    padding: 10px;
    margin: 0;
} 

.ContactsPage .Map{
    width: 100%;
}


@media all and (max-width: 790px) {

}

