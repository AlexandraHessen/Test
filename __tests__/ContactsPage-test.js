"use strict";

import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';

import ContactsPage from '../pages/ContactsPage';
                    //импортируем файл который будем тестировать

test('ContactsPage', () => {
                            // создаём тестовую версию компонента который будем тестировать
  const component = renderer.create(
    <BrowserRouter>
      <ContactsPage />
    </BrowserRouter>
  );

                            // получаем снэпшот (HTML-снимок) компонента для сверки, чтобы вёрстка не испортилась
  let componentTree=component.toJSON();
  expect(componentTree).toMatchSnapshot();

    
});
