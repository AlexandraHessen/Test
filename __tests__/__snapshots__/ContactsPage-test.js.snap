// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactsPage 1`] = `
<div
  className="ContactsPage"
>
  <h1>
    Контакты
  </h1>
  <h3>
    Наш адрес:
  </h3>
  <p>
    г.Минск, пр.Победителей, 65
  </p>
  <h3>
    Телефон:
  </h3>
  <p>
    +375 (29) 844-44-88
  </p>
  <p>
    +375 (44) 888-44-88
  </p>
  <h3>
    E-mail:
  </h3>
  <p>
    plants@gmail.com
  </p>
  <h3>
    Карта проезда:
  </h3>
  <iframe
    allowFullScreen={true}
    className="Map"
    frameBorder="0"
    height="450"
    src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2349.3038246568763!2d27.515363315307575!3d53.92634593880996!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x46dbc56e350faf59%3A0x8039bcd46f24618a!2z0L_RgC3Rgi4g0J_QvtCx0LXQtNC40YLQtdC70LXQuSA2NSwg0JzQuNC90YHQug!5e0!3m2!1sru!2sby!4v1638823143292!5m2!1sru!2sby"
    width="800"
  />
</div>
`;
